src/main/java/de/False/BuildersWand/events/WandEvents.java

```diff
@@ -520,7 +520,9 @@ private void renderBlockOutlines(BlockFace blockFace, Block selectionBlock, List
     private boolean isAllowedToBuildForExternalPlugins(Player player, Location location) {
         Plugin townyPlugin = getExternalPlugin("Towny");
         if (townyPlugin != null) {
-            return PlayerCacheUtil.getCachePermission(player, location, Material.STONE, TownyPermission.ActionType.BUILD);
+            if (PlayerCacheUtil.getCachePermission(player, location, Material.STONE, TownyPermission.ActionType.BUILD)) {
```

Comment:
This should be inverted i guess, before it would return now it would false.

Reply:
Oh shoot, you're right.